src/module/settings.ts

```diff
@@ -62,6 +62,8 @@ export const registerSettings = function ():void {
   _booleanSetting('showHeroPoints', false);
   _booleanSetting('showSingleComponentColumn', false);
 
+  _numberSetting('weightModifierForWornArmor', 0.25);
```

Comment:
Set the default to 1 please so it matches Cepheus Engine/Deluxe rules (as that's what we have the license to use.)
You might also want to change the MGT 2 section in docs/docs.md so that it mentions that MGT2 users should change this to 0.25

Reply:
Never mind. Come to think of it, I'll just change it myself when I do the release in a little while.